knowledge-graph/src/main/scala/io/renku/knowledgegraph/docs/Endpoint.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.knowledgegraph.docs
+
+import cats.effect.Async
+import cats.implicits.catsSyntaxApplicativeId
+import cats.syntax.all._
+import io.circe.syntax._
+import io.renku.knowledgegraph.docs.Implicits.StatusOps
+import io.renku.knowledgegraph.docs.model._
+import io.renku.knowledgegraph.lineage
+import org.http4s
+import org.http4s.circe.jsonEncoder
+import org.http4s.dsl.Http4sDsl
+
+trait Endpoint[F[_]] {
+  def `get /docs`: F[http4s.Response[F]]
+}
+
+private class EndpointImpl[F[_]: Async] extends Http4sDsl[F] with Endpoint[F] {
+  import Encoders._
+
+  override def `get /docs`: F[http4s.Response[F]] = Ok(doc.asJson)
+
+  lazy val doc: OpenApiDocument =
+    OpenApiDocument(
+      "3.0.3",
+      Info("Knowledge Graph API", "Get info about datasets, users, activities, and other entities".some, "1.0.0")
+    ).addServer(renkuLabServer)
+      .addPath(lineage.EndpointDoc.path)
+      .addSecurity(securityScheme)
```

Comment:
Would that mean that we'll have only one security scheme in the docs? It would be nice if we can add these three: oauth, private token and no auth.